app/frontend/components/pages/delegations/deregisterStakeKey.tsx

```diff
@@ -21,11 +15,7 @@ const DeregisterStakeKeyPage = () => {
       <p className="deregister-stake-key-card-disclaimer">
         ...if you do not want to use this wallet anymore
       </p>
-      <button
-        className="button secondary cancel-delegation"
-        disabled={calculatingDelegationFee}
-        onClick={() => deregisterStakingKey()}
-      >
+      <button className="button secondary cancel-delegation" onClick={() => deregisterStakingKey()}>
```

Comment:
If we use the new approach for multiple transactions we can remove such checks from more places.